pkg/deploy/api/types.go

```diff
@@ -257,6 +257,8 @@ type DeploymentConfig struct {
 	// Details are the reasons for the update to this deployment config.
 	// This could be based on a change made by the user or caused by an automatic trigger
 	Details *DeploymentDetails
+	// Replicas is the total number of pods targeted by this deploymentConfig.
+	Replicas int
```

Comment:
OK, I think I want to change this here. Refactor the internal representation to use status and spec, add a status subresource and a separate call.